Makefile

```diff
@@ -119,18 +119,21 @@ uninstall:
 .PHONY: uninstall
 
 opm:
-	@if [ ! -f "${GOPATH}/src/github.com/operator-framework/operator-registry/opm" ] ; then \
+	@if [ ! -f "${GOPATH}/src/github.com/operator-framework/operator-registry/bin/opm" ] ; then \
 		mkdir -p ${GOPATH}/src/github.com/operator-framework ||: ; \
 		pushd ${GOPATH}/src/github.com/operator-framework ; \
 		[ ! -d ./operator-registry ] && git clone https://github.com/operator-framework/operator-registry ; \
 		cd operator-registry ; \
-		go build ./cmd/opm/ ; \
+		make build ; \
 		popd ; \
 	fi
 .PHONY: opm
 
+MANIFEST_VERSION?="4.6"
 generate-bundle: opm
```

Comment:
I suggest to make `regenerate` a target dependency to ensure that the bundles picks always the latest CRDs.